utils.js

```diff
@@ -16,7 +16,7 @@ async function getSelfUser() {
 
 async function checkUserIsSuperUser() {
   const self_user = await getSelfUser();
-  return self_user?.roles['super_user'];
+  return self_user?.roles['super_user'] || false;
 }
```

Comment:
why are we returning false from here?

Reply:
my bad I will undo this change i just thought instead of returning undefined let it return false would look better.